resources/web/style/heading.pcss

```diff
@@ -1,4 +1,23 @@
 #guide {
+
+  div.heading {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    /* Give all headings plenty of space above them */
+    margin-top: 40px;
+    margin-bottom: 10px;
+
+    a, h1, h2, h3, h4, h5, h6 {
+      display: flex
+    }
```

Comment:
What is the benefit of employing flexbox here? It's within a pretty small scope so I guess introducing accidental flexbox bugs is probably unlikely but I'm just curious. I've learned just enough flexbox to be equal parts dangerous and wary of its ways. 😄

Reply:
It's the best way **I know** to have two elements center-aligned vertically, while having the second one pushed all the way to the right. That's not the same as it being the _actual_ best way, though. 😁 

I started trying by trying to set the `.edit_me` icons as `float: right`, but the vertical alignment was a mess, and I was trying to minimize the changes to HTML (I  basically just added the `heading` class to the enclosing `<div>` to make it easier to target with this rule.

Reply:
Totally fair. `display: flex` is the new `float:` 😄